tests/CMakeLists.txt

```diff
@@ -126,6 +126,27 @@ if (GGML_OPENBLAS)
     endif()
 endif()
 
+if (GGML_CLBLAST)
+        set(CLBLAST_INCLUDE_SEARCH_PATHS
+        /usr/include
+        /usr/local/include
+        $ENV{CLBLAST_HOME}
+        $ENV{CLBLAST_HOME}/include
+        )
+        find_path(CLBLAST_INC NAMES cblast.h PATHS ${CLBLAST_INCLUDE_SEARCH_PATHS})
+        find_library(CLBLAST_LIB NAMES clblast)
+        if (CLBLAST_LIB)
+                message(STATUS "clBLAST found")
+
+                set(GGML_EXTRA_LIBS  ${GGML_EXTRA_LIBS}  ${CLBLAST_LIB})
+		set(GGML_OPENCL_SOURCES ggml-opencl.cpp ggml-opencl.h)
+                set(GGML_EXTRA_FLAGS ${GGML_EXTRA_FLAGS} -DGGML_USE_CLBLAST)
+		link_libraries("-Wl,--copy-dt-needed-entries")
+    else()
+        message(WARNING "clBLAST not found")
+    endif()
+endif()
+
```

Comment:
Is this needed given that we added CLBLAST to the `ggml` target in the root CMakeLists.txt